Packs/CarbonBlackDefense/Integrations/CarbonBlackLiveResponseCloud/CarbonBlackLiveResponseCloud.py

```diff
@@ -0,0 +1,319 @@
+                                      removeNull=True)
+
+    return CommandResults(
+        outputs_prefix='CarbonBlackDefenseLR.RegistryValues',
+        outputs_key_field='sensor_id',
+        outputs=context_entry,
+        readable_output=readable_output,
+        raw_response=values
+    )
+
+
+# Using Py API
+def create_reg_key_command(credentials: Dict, sensor_id: str, reg_path: str):
+    api = CBCloudAPI(**credentials)
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    session.create_registry_key(reg_path)
+    return f'Reg key: {reg_path}, was created'
+
+
+# Using Py API
+# TODO check default params of overwrite in the YML
+def set_reg_value_command(
+        credentials: Dict, sensor_id: str, reg_path: str,
+        value_data: Any, value_type: str = None, overwrite: bool = True):
+    api = CBCloudAPI(**credentials)
+
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    session.set_registry_value(reg_path, value_data, overwrite=overwrite, value_type=value_type)
+    return f'Value was set to the reg key: {reg_path}'
+
+
+# Using Py API
+def delete_reg_key_command(credentials: Dict, sensor_id: str, reg_path: str):
+    api = CBCloudAPI(**credentials)
+
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    session.delete_registry_key(reg_path)
+    return f'Registry key: {reg_path} was deleted'
+
+
+# Using Py API
+def delete_reg_value_command(credentials: Dict, sensor_id: int, reg_path: str):
+    api = CBCloudAPI(**credentials)
+
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    session.delete_registry_value(reg_path)
+    return f'Registry value: {reg_path} was deleted'
+
+
+# Using Py API
+def list_processes_command(credentials: Dict, sensor_id: int):
+    api = CBCloudAPI(**credentials)
+
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    processes = session.list_processes()
+    if not processes:
+        return 'There is no active processes'
+
+    headers = ['path', 'pid', 'command_line', 'username']
+    processes_readable = [dict(
+        path=process['path'],
+        pid=process['pid'],
+        command_line=process['command_line'],
+        user_name=process['username']) for process in processes]
+    context_entry = dict(sensor_id=sensor_id, processes=processes)
+
+    readable_output = tableToMarkdown('Carbon Black Defense Live Response Processes',
+                                      headers=headers,
+                                      t=processes_readable,
+                                      headerTransform=string_to_table_header,
+                                      removeNull=True)
+    return CommandResults(
+        'CarbonBlackDefenseLR.Processes',
+        outputs_key_field='sensor_id',
+        outputs=context_entry,
+        readable_output=readable_output,
+        raw_response=processes
```

Comment:
```suggestion
        raw_response=processes,
```

Fix this in all relevant places as we discussed regarding the PEP8